Makefile

```
@@ -29,3 +29,9 @@ build:
 
 run:
 	go run $(MODULE)/...
+
+check-vendor: vendor
```

Nit: Can we rename this:
```suggestion
check-modules: vendor
```